documentation/restler/QuickStart.md

```diff
@@ -0,0 +1,139 @@
+# Quick start with RESTler API testing tool
+
+## Prerequisites
+
+Install [Python 3.8.2][] and [.NET 6.0][] or higher, for your appropriate OS.
+
+[Python 3.8.2]: https://www.python.org/downloads/
+[.NET 6.0]:https://dotnet.microsoft.com/download/dotnet-core?utm_source=getdotnetcorecli&utm_medium=referral
+
+## Install
+
+Here are the steps to build and install RESTler locally.
+See the [RESTler project readme][] for other installation options.
+
+[RESTler project readme]: https://github.com/microsoft/restler-fuzzer/blob/main/README.md
+
+```sh
+git clone https://github.com/microsoft/restler-fuzzer.git
+restler_bin=~/bin/restler
+mkdir -p $restler_bin
+# Need --python because it can’t find “python” — I use an alias that it did not understand.
+python ./build-restler.py --dest_dir $restler_bin --python_path /opt/homebrew/bin/python3
+```
+
+## Create the initial Restler config and dict files
+
+You need to create a Restler config file and a dict file for each API you want to test. You can use RESTler to create the initial files for you, with the `generateConfig` command. You need to specify the API spec file(s) -- here we use the Azure Databricks service as an example. 
+
+```sh
+specs=$(find /Users/mikekistler/Projects/Azure/azure-rest-api-specs/specification/databricks/resource-manager/Microsoft.Databricks/preview/2022-04-01-preview -type f -depth 1)
+dotnet $restler_bin/restler/Restler.dll generate_config --specs ${=specs}
+```
+
+This creates a new directory `restlerConfig` with the following files:
+- `annotations.json` - an empty file where you can add annotations for the API spec
+- `config.json` - the Restler config file
+- `dict.json` - a Restler dictionary file
+- `engine_settings.json` - the Restler engine settings file  
+
+## Customize the Restler configuration files
+
+You often need to specify values for variables in the API definition.
+You can do this by adding these values in `restler_custom_payload` property in the `dict.json` file.
+
+For ARM APIs you generally need to specify the subscriptionId, resourceGroupName, and location.
+You may also want to specify the api-version, in case an operation does not have an example value for it.
+You can use the `jq` command to add these values. Here is an example:
+
+```sh
+vars='{
+    "subscriptionId": ["my-subscription-id"],
+    "resourceGroupName": ["my-resource-group"],
+    "location": ["my-location"],
+    "api-version": ["2022-04-01-preview"]
+}'
+jq ".restler_custom_payload = $vars" restlerConfig/dict.json > dict.json
+```
+
+Next update `config.json` to disable data fuzzing.
+
+```sh
+jq '.DataFuzzing = false' restlerConfig/config.json > config.json
+```
```

Comment:
Why?

Reply:
FYI, this will cause there to be fewer dependencies, so I recommend leaving this in unless it is causing problems.

Reply:
I added this early on when I was having problems but then had to fix other things as well, so this change may not actually be needed.  I will remove it for now.